src/src/com/microsoft/aad/adal/DefaultTokenCacheStore.java

```diff
@@ -112,16 +113,14 @@ private String encrypt(String value) {
         return null;
     }
 
-    private String decrypt(String value) {
+    private String decrypt(final String key, String value) {
```

Comment:
make value as final too. Consider updating eclipse's rules to handle all the formatting for you.